# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../sn9c102/linux-2.6.18.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./sn9c102_core.c.orig	2006-10-06 17:11:58.000000000 -0400
+++ ./sn9c102_core.c	2006-10-06 17:13:10.000000000 -0400
@@ -39,6 +39,8 @@
 #include <asm/page.h>
 #include <asm/uaccess.h>
 
+#define __OLD_VIDIOC_
+
 #include "sn9c102.h"
 
 /*****************************************************************************/
